mqtt/mqtt-bridge/config/default.json

```diff
@@ -2,8 +2,16 @@
     "upstream": {
         "keep_alive": "1m",
         "clean_session": false,
-        "subscriptions": [],
-        "forwards": []
+        "forwards": [
```

Comment:
this should be reverted

Reply:
Yes I am going to keep it here until I do a final test.